jawanndenn/static/html/poll.xhtml

```diff
@@ -7,15 +7,16 @@ Licensed under GNU GPL v3 or later
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
     <head>
         <title>jawanndenn: Vote!</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        <script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.min.js" />
-        <script src="../static/3rdparty/noty-2.4.1/js/noty/packaged/jquery.noty.packaged.min.js" />
+        <script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.min.js" ></script>
```

Comment:
These changes make the templates compatible with standard html - the layout is broken with the standard backend otherwise